binary/test_suite/double_bh/inlist

```diff
@@ -17,9 +17,9 @@
 
 
 
-&binary_pgstar
+&pgbinary
 
-      read_extra_binary_pgstar_inlist1 = .true.
-      extra_binary_pgstar_inlist1_name = 'inlist_project'
+      read_extra_pgbinary_inlist1 = .true.
```

Comment:
Would you mind adding some examples in the a test case (with the default pgbinary_flag=.false.) That way people are more likely to notice there are now plot options.

Reply:
absolutely!